frontend/src/components/filterPage/FilterPage.tsx

```diff
@@ -0,0 +1,69 @@
+import "./FilterPage.css";
+import {ChangeEvent, Dispatch, FormEvent, SetStateAction, useState} from "react";
+import {formatEnum} from "../../utils/utilFunctions.ts";
+
+type FilterPageProps = {
+    selectedGenre: string,
+    setSelectedGenre: (genre: string) => void,
+    handleApplyFilter: (genre: string) => void,
+    setShowKeywordTag: Dispatch<SetStateAction<boolean>>
+}
+
+export default function FilterPage({ setSelectedGenre, handleApplyFilter, setShowKeywordTag }: FilterPageProps) {
+    const [isDisable, setIsDisable] = useState<boolean>(true);
+    const [genre, setGenre] = useState<string>('Select');
+
+    const genres : string[] = ["NONE", "FICTION", "MYSTERY", "THRILLER",
```

Comment:
```suggestion
    const genres : Genre[] = ["NONE", "FICTION", "MYSTERY", "THRILLER",
```
created a new Type Genre that we can use for this

Reply:
erledgit